### DISCUSSION TITLE
A205 Carrier Board | Seeed Studio Wiki

### DISCUSSION THREAD

**Discussion by giscus** [Seeed-Studio/wiki-documents#85] [Q&A]
# A205 Carrier Board | Seeed Studio Wiki

A205 Carrier Board

https://wiki.seeedstudio.com/reComputer_A205_Flash_System/#docusaurus_skipToContent_fallback

**Comment by sealfoss** [comment]
What battery do you need for A205 CMOS?

**Reply by MatthewJeffson** [reply]
This I can help you:D, it's CR1220

**Comment by sealfoss** [comment]
Can we get drivers for Jetpack 5.1.1?

**Comment by yemvibol** [comment]
Will it work with IMX708 12MP camera?

**Comment by yemvibol** [comment]
Do you know how to test CSI camera? I flashed JetPack OS L4T version 35.0.1 via Command Line sucessfully, then I connected raspberry pi camera V2.1 (should be IMX-219) and used v4l2 command to check but the camera was not found.

**Reply by yemvibol** [reply]
it works after second time try

**Reply by MatthewJeffson** [reply]
Hi! Glad to hear that! Thanks for interesting our products! :D

**Reply by yemvibol** [reply]
Well, I like it. By the way, Flash System on SD card is not yet released, right? I clicked the above link but it is not working.

**Comment by realtime3392** [comment]
Should a micro SD card inserted into the micro SD card slot on the A205 shouw up in the output of `lsblk`?

I only see the 16GB eMMC of my Xavier NX 8GB module, as /dev/mmcblk0 in the `lsblk` output.

Thanks!

**Comment by neoscell** [comment]
Can we get driver update for Jetpack 6? Need this to use latest packages with Jetson and this board please!

**Reply by nfs0619** [reply]
Thank you for reaching out. Unfortunately, the modules supported by the A205 carrier board can only be upgraded to JetPack 5.1.4 at most.

For a full list of JetPack versions and supported modules, please refer to the official JetPack Archive here:
[JetPack Archive](https://developer.nvidia.com/embedded/jetpack-archive)

If you need further assistance or have any more questions, feel free to let us know!
